service/collector/helper.go

```diff
@@ -183,3 +217,58 @@ func (h *helper) AWSAccountID(awsClients clientaws.Clients) (string, error) {
 
 	return accountID, nil
 }
+
+// IsClusterReconciledByThisVersion checks whether an AWSCluster/AWSConfig
+// object with the given name is reconciled by this particular version of
+// aws-operator (based on version found in pkg/project/project.go).
+func (h *helper) IsClusterReconciledByThisVersion(clusterName string) (bool, error) {
+	if clusterName == "" {
+		return false, nil
+	}
+	key := fmt.Sprintf("default/%s", clusterName)
+
+	var (
+		item   interface{}
+		exists bool
+		err    error
+	)
+
+	for _, store := range []cache.Store{h.awsClustersCache, h.awsConfigsCache} {
+		item, exists, err = store.GetByKey(key)
+		if err != nil {
+			return false, microerror.Mask(err)
+		}
+		if exists {
+			break
+		}
+	}
+	if !exists {
+		h.logger.Log(
+			"level", "warning",
```

Comment:
Debug at best IMO. This will happen all the time. I'd skip this log altogether.

```suggestion
			"level", "debug",
```